spec.html

```diff
@@ -24990,10 +24994,13 @@ <h1>CreateDynamicFunction ( _constructor_, _newTarget_, _kind_, _args_ )</h1>
             1. Let _prefix_ be the prefix associated with _kind_ in <emu-xref href="#table-dynamic-function-sourcetext-prefixes"></emu-xref>.
             1. Let _sourceString_ be the string-concatenation of _prefix_, *" anonymous("*, _P_, 0x000A (LINE FEED), *") {"*, _bodyString_, and *"}"*.
             1. Let _sourceText_ be ! StringToCodePoints(_sourceString_).
+            1. Let _parameters_ be ParseText(! StringToCodePoints(_P_), _parameterSym_).
+            1. If _parameters_ is a List of errors, throw a *SyntaxError* exception.
+            1. Let _body_ be ParseText(! StringToCodePoints(_bodyString_), _bodySym_).
+            1. If _body_ is a List of errors, throw a *SyntaxError* exception.
             1. Let _expr_ be ParseText(_sourceText_, _exprSym_).
             1. If _expr_ is a List of errors, throw a *SyntaxError* exception.
-            1. Let _parameters_ be the |FormalParameters| of _expr_.
-            1. Let _body_ be the child of _expr_ that is an instance of _bodySym_.
+            1. NOTE: The above two steps ensure that Early Error rules which apply to _exprSym_ directly are enforced.
```

Comment:
I'd like this note to be even more explicit about a reparse being unnecessary.

Reply:
Rephrased to

>             1. NOTE: If the above two steps are reached, _sourceText_ is guaranteed to match _exprSym_. The purpose of these steps is to enforce any Early Error rules which apply to _exprSym_ directly.

Does that work for you?

Reply:
Works for me, thanks.